client/lib/platform/index.js

```diff
@@ -0,0 +1,18 @@
+export const ANDROID_PLATFORM = 'android';
+export const IOS_PLATFORM = 'ios';
+export const UNKNOWN_PLATFORM = 'unknown';
+
+/**
+ * Provide basic detection of platform based on user agent. This is not
+ * a robust check for browser features or the like. You should only use
+ * this for non-critical display logic.
+ */
+export const platform = () => {
```

Comment:
Basic UA string checking for platform based on other faux smart app banner approaches I've seen.